runelite-client/src/main/java/net/runelite/client/plugins/zoom/ZoomPlugin.java

```diff
@@ -164,6 +176,23 @@ public void keyReleased(KeyEvent e)
 		if (e.getKeyCode() == KeyEvent.VK_CONTROL)
 		{
 			controlDown = false;
+
+			if (zoomConfig.controlFunction() == ControlFunction.CONTROL_TO_RESET)
+			{
+				int finalZoomValue;
+				int zoomValue = zoomConfig.ctrlZoomValue();
```

Comment:
Please just do a normal bounds check.

Reply:
So you're saying I shouldn't clamp the value and allow the zoom reset to occur if it's out of bounds?

Reply:
`zoomValue` is mutable. `finalZoomValue` has no purpose and should not exist.